src/Bot.ts

```diff
@@ -91,6 +90,23 @@ export class Bot extends builder.UniversalBot {
         };
     }
 
+    private SigninHandler(bot: builder.UniversalBot): (event: builder.IEvent, query: teams.ISigninStateVerificationQuery, callback: (err: Error, body: any, status?: number) => void) => void {
+        return async function (
```

Comment:
We should change this getXXXHandler() pattern as it's not clear how to access private members in the handler (this confused Shoeb who was working on the Trello CE).

Better to do something like this:
    this._connector.onSigninStateVerification((event, query, callback) => { this.verifySigninState(event, query, callback); });
and put your logic in the private member function verifySigninState().

Reply:
Done.